src/scripts/main.js

```diff
@@ -2,7 +2,20 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
+let count = 1;
 
-// write your code here
+people.forEach(person => {
+  table.insertRow();
+
+  const row = document.querySelectorAll('tr');
+
+  row[count].insertCell(0).innerText = person.name;
+  row[count].insertCell(1).innerText = person.sex;
+  row[count].insertCell(2).innerText = person.born;
+  row[count].insertCell(3).innerText = person.died;
+  row[count].insertCell(4).innerText = person.died - person.born;
+  row[count].insertCell(5).innerText = Math.ceil(person.died / 100);
+
+  count++;
```

Comment:
![Screenshot](https://user-images.githubusercontent.com/69436963/195319930-e0a09e0f-a9d3-41ea-93fc-e5576c9fc537.png)
I must have made some mistake because I wanted to delete this Pull requests.
I made these corrections in another
https://github.com/mate-academy/js_task_generate_table_DOM/pull/532